src/modules/Footer/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { SectionContainer } from "../../components/General.styled";
+import Menu from "./Menu/Menu";
+import BottomFooter from "./Social/BottomFooter";
+import Social from "./Social/Social";
+
+const Footer = () => {
+  return (
+    <Container>
+      <TopContainer>
+        <Social />
+        <Menu />
+      </TopContainer>
+      <BottomFooter />
+    </Container>
+  );
+};
+
+export default Footer;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TopContainer = styled(SectionContainer)`
+  display: flex;
```

Comment:
display: flex; -> is redundant
TopContainer inherited flex from SectionContainer
Please remove it.